examples/python-kernel-wrapper/cython/setup.py

```diff
@@ -0,0 +1,108 @@
+import os
+import shutil
+import sysconfig
+from distutils.sysconfig import get_python_lib
+
+import numpy
+from Cython.Distutils import build_ext
+from setuptools import Extension, find_packages, setup
+
+# Locate CUDA_HOME
+CUDA_HOME = os.environ.get("CUDA_HOME", False)
+if not CUDA_HOME:
+    path_to_cuda_gdb = shutil.which("cuda-gdb")
+    if path_to_cuda_gdb is None:
+        raise OSError(
+            "Could not locate CUDA. "
+            "Please set the environment variable "
+            "CUDA_HOME to the path to the CUDA installation "
+            "and try again."
+        )
+    CUDA_HOME = os.path.dirname(os.path.dirname(path_to_cuda_gdb))
+
+cuda_include_dir = os.path.join(CUDA_HOME, "include")
+cuda_lib_dir = os.path.join(CUDA_HOME, "lib64")
```

Comment:
I don't think you need to find / include CUDA here since the Cython you've written doesn't directly depend on CUDA in any way. (I could be mistaken though in just cimporting cudf things could trigger the dependency).